types/finding.ts

```diff
@@ -11,6 +11,8 @@ export interface Finding {
   updatedAt: string;
   issueNumber: number;
   handle: string;
+  isMitigated?: boolean;
+  mitigationOf?: string;
```

Comment:
Should it be defined as you defined it below ??  
```
 mitigationOf?: {
    newValue: string;
    oldValue: string;
  };
  ```

Reply:
No, the interface definition below is for a request to edit a finding, so it has the previous value and the new value. This is the generic finding interface